scripts/funcTests/runFuncTests.sh

```diff
@@ -68,6 +68,8 @@ do
 done
 echo "================="
 
+# install SDK2 runtime as we encounter problems on running dotnet vstest command when only download SDK3.
+cli/dotnet-install.sh -runtime dotnet --version 2.2.3 -i cli -NoPath
```

Comment:
as my previous comment, I don't think this is necessary as we should be targeting 3.0 for all our tests.

Reply:
As discussed offline, ignore this comment.